src/app/app.component.html

```diff
@@ -1,9 +1,10 @@
 <div class="ms-Grid"> 
     <div class="ms-Grid-row">
-    <sidebar class="ms-Grid-col ms-u-sm12 ms-u-md12 ms-u-lg4 ms-u-xl3 ms-u-xxl3 ms-u-xxxl2"></sidebar>
-    <main-column class="ms-Grid-col ms-u-sm12 ms-u-md12  ms-u-lg8 ms-u-xl9 ms-u-xxl9 ms-u-xxxl10" id="explorer-main"></main-column>
+      <sidebar class="ms-Grid-col ms-u-sm12 ms-u-md12 ms-u-lg4 ms-u-xl3 ms-u-xxl3 ms-u-xxxl2"></sidebar>
+      <main-column class="ms-Grid-col ms-u-sm12 ms-u-md12  ms-u-lg8 ms-u-xl9 ms-u-xxl9 ms-u-xxxl10" id="explorer-main"></main-column>
+    </div>
```

Comment:
I'm sitting here, staring at this change asking myself, is it a whitespace change? is it just Git that is confused? Is there actually a change here? What does this have to do with Angular CLI?

Reply:
Another false positive. Whitespace change.